zosia_site/README.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute?
+
+## 1. Make sure that your Google account has admin access to the Zosia GCP project
+
+## 2. Install terraform and gcloud locally
+
+Run these commands:
+
+`gcloud config set project <project_id>` - sets the default project for your gcloud client
+
+`gcloud auth application-default login` - obtains your ADC (Application Default Credentials), which terraform will later use automatically to authenticate with Google Cloud
+
+## 3. Modify terraform code
+
+## 4. Run `terraform plan`. If you're happy with the results, run `terraform apply`
+
+## 5. Run `terraform fmt` and create a Pull Request with changes
+
+# How to deploy this infrastructure to a new Google Cloud account?
+
+## 1. Create a GCP project and save project id
+
+## 2. Install terraform and gcloud locally
+
+Run these commands:
+
+`gcloud config set project <project_id>` - sets the default project for your gcloud client
+
+`gcloud auth application-default login` - obtains your ADC (Application Default Credentials), which terraform will later use automatically to authenticate with Google Cloud
+
+`gcloud services enable storage.googleapis.com secretmanager.googleapis.com appenginereporting.googleapis.com sql-component.googleapis.com` - enables required services in your project
+
```

Comment:
Technically, you can add this to terraform code as well using [google_project_service](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_service) resource.

Reply:
Done